src/handlebarHelpers.ts

```diff
@@ -56,6 +56,9 @@ Handlebars.registerHelper('refToResourceLink', (ref) => {
 })
 
 function refToResourceName(ref: string): string | undefined {
+    if (ref === undefined) {
```

Comment:
`ref` can't be undefined if we're to trust the function signature, should we change the signature here?

Reply:
yup